trajopt_ext/qpoases/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+cmake_minimum_required(VERSION 3.2.1)
+find_package(ros_industrial_cmake_boilerplate REQUIRED)
+
+extract_package_metadata(pkg)
+project(qpoases VERSION ${pkg_extracted_version} LANGUAGES CXX)
+
+find_package(qpoases QUIET)
```

Comment:
This needs to be `qpOASES` instead of `qpoases`, if not it builds qpOASES everytime.

Reply:
Fixed